### ISSUE TITLE
UI for advanced repeat rules

### ISSUE THREAD

**Issue by zenhack** [zenhack/sandcal#1]
Right now the UI only provides fairly simple controls over the recurrence of events: you can choose a frequency, (daily, weekly...) and that's it. We should provide more advanced options. In particular, there are some things we already support on the backend (so they can be used if you import an ics file) but don't have a UI for:

- [ ] The 'interval' option, so you can say things like "every other week'.
- [ ] Multiple recurrence rules for a single event.

There are many other things supported by the iCalendar format; we should ideally implement all of them.

**Comment by ocdtrekkie** [comment]
Yeah, I looked for this immediately upon testing. I imagine the easiest way to make a recurring Office Hours is two recurring events, each set for every other week.

**Comment by zenhack** [comment]
That has the drawback that a potential delete/edit option would not know that they are the "same" event, and the logic for handling already-stored events with multiple recurrence rules is already there. Agree it's a reasonable workaround, but it would be nice to have an actual UI for the feature nonetheless.

**Comment by zenhack** [comment]
I've implemented UIs for the two specific features above. There are still a bunch of ics features we don't support though, either in terms of UI or just in terms of interpreting them.

**Comment by ocdtrekkie** [comment]
My current need is to represent every third Tuesday of the month. I also do not see the interval option described above.

**Comment by zenhack** [comment]
"interval" is what it's called in the iCalendar spec, but in sandcal's ui it's the numeric field in `Every __ days/weeks/etc`

**Comment by zenhack** [comment]
Correct. That's not just a UI issue, the backend needs to support the logic too.
